File: fundamentals/easy/countByX.ts
function countByX(x: number, n: number) {
	// const z: number[] = []

	// for (let i = x; i <= x * n; i += x) z.push(i)

	// return z

	Array.from({ length: n }, (_, key) => key + 1 * x)
}

export { countByX }

File: codewars/julia/easy/countByX.jl
function countByX(num::Int, multiple::Int)
  count = num
  resultArr::Array{Int,1} = []

  while multiple > 0
    push!(resultArr, count)
    count += num
    multiple -= 1
  end

  return resultArr
end

println(countByX(1, 5)) # [1, 2, 3, 4, 5]
println(countByX(2, 5)) # [2, 4, 6, 8, 10]